src/lighteval/commands/lighteval_cli.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+# MIT License
+
+# Copyright (c) 2024 Taratra D. RAHARISON and The HuggingFace Team
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import argparse
+import importlib
+import json
+import os
+
+import pkg_resources
+
+
+def load_tasks_table_extended(module_name: any) -> list:
+    """
+    load the module module_name
+
+    Args:
+    - module_name the name of the module we want to load
+    Returns:
+    - TASKS_TABLE: a list of the task in the module
+    """
+    module_path = f"lighteval.tasks.extended.{module_name}.main"
+    module_loaded = importlib.import_module(module_path)
+    tasks_list = None
+    try:
+        tasks_list = module_loaded.TASKS_TABLE
+    except Exception as e:
+        print(e)
+    return tasks_list if tasks_list is not None else []
+
+def getTasksTableJson()->list:
```

Comment:
Please use Python style for casing: ClassName, function_name

Reply:
Okay. I changed the casing.